src/main/kotlin/no/nav/familie/baks/mottak/task/FinnmarkstilleggTask.kt

```diff
@@ -46,7 +48,18 @@ class FinnmarkstilleggTask(
                 !sisteBostedsadresse.erIFinnmarkEllerNordTroms() &&
                 nestSisteBostedsadresse.erIFinnmarkEllerNordTroms()
 
-        task.metadata["harFlyttet"] = if (harFlyttetInnEllerUtAvFinnmarkEllerNordTroms) "Ja" else "Nei"
+        if (harFlyttetInnEllerUtAvFinnmarkEllerNordTroms) {
```

Comment:
Som kommentert i ba-sak, så tror jeg at logikken i harFlyttetInnEllerUtAvFinnmarkEllerNordTroms ikke vil dekke alle tilfellene vi ønsker å sende til ba-sak.
https://github.com/navikt/familie-ba-sak/pull/5502/files#r2262163429

Reply:
I ba-sak kan vi jo muligens legge inn logikk for å sjekke om forrige iverksatte behandling hadde finnmarkstillegg eller ikke for personen. I såfall kanskje vi kan endre logikken i filtreringen her i baks-mottak til at vi sender melding dersom nåværende adresse er i Finnmark/Nord-Troms eller man har bodd i Finnmark/Nord-Troms tidligere? 🤔